src/sonic-host-services/scripts/determine-reboot-cause

```diff
@@ -174,7 +174,7 @@ def main():
         os.makedirs(REBOOT_CAUSE_DIR)
 
     # Remove stale PREVIOUS_REBOOT_CAUSE_FILE if it exists
-    if os.path.exists(PREVIOUS_REBOOT_CAUSE_FILE):
+    if os.path.exists(PREVIOUS_REBOOT_CAUSE_FILE) or os.path.islink(PREVIOUS_REBOOT_CAUSE_FILE):
```

Comment:
this change doesn't seem to make any difference.

Reply:
@mlok-nokia seems it is reverted but looks like we need this. Can you create PR ? wondering do you not see issue even though it is reverted ?

cc @judyjoseph for reference.